src/apollo/mutations.ts

```diff
@@ -1,23 +1,50 @@
-// import gql from "@apollo/client";
+import { gql } from "@apollo/client";
 
-// import { accountErrorFragment, checkoutErrorFragment, paymentErrorFragment, checkoutFragment, orderDetailFragment, paymentFragment, userFragment } from "./fragments";
+import {
+  accountErrorFragment,
+  // checkoutErrorFragment,
+  // paymentErrorFragment,
+  // checkoutFragment,
+  // orderDetailFragment,
+  // paymentFragment,
+  // userFragment,
+} from "./fragments";
 
-// export const tokenAuthMutation = gql`
-//   ${accountErrorFragment}
-//   mutation TokenAuth($email: String!, $password: String!) {
-//     tokenCreate(email: $email, password: $password) {
-//       csrfToken
-//       refreshToken
-//       token
-//       errors: accountErrors {
-//         ...AccountError
-//       }
-//       user {
-//         id
-//       }
-//     }
-//   }
-// `;
+export const LOGIN = gql`
+  ${accountErrorFragment}
+  mutation login($email: String!, $password: String!) {
```

Comment:
Naming convention
```suggestion
export const loginMutation = gql`
  ${accountErrorFragment}
  mutation loginMutation($email: String!, $password: String!) {
```

Reply:
Here I don't have a strong opinion: the mutation and queries will be imported from `/mutations` or `/queries`, so it will be easy to see what type of action it is. IMO the shorter and also meaningful name, the better but let me know what is the final decision and I will change it.

Reply:
I agree with @karolinakuzniewicz here, I don't see a reason to append `mutation/query` to the name when they're defined in a file name that defines the action

Reply:
also: capital letters indicate that this is some constant definition and not a function itself, where `loginMutation` could be mistaken for a function that calls the mutation